### PR TITLE
[New Rule] Remote File Copy via TeamViewer

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/240

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/command_and_control_teamviewer_remote_file_copy.toml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+risk_score = 47
+rule_id = "b25a7df2-120a-4db2-bd3f-3e4b86b24bee"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:file and event.type:creation and
+ process.name:TeamViewer.exe and
+ file.name:(*.exe or *.dll or *.scr or *.com or *.bat or *.ps1 or *.vbs or *.vbe)
```
Comment (brokensound77) [inline rules/windows/command_and_control_teamviewer_remote_file_copy.toml:27]: should be able to use file.extension here instead

Comment (Samirbous) [reply rules/windows/command_and_control_teamviewer_remote_file_copy.toml]: converted

File: rules/windows/command_and_control_teamviewer_remote_file_copy.toml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an executable or script file remotely downloaded via a TeamViewer transfer session, verify the context of
+this activity.
```
Comment (bm11100) [inline rules/windows/command_and_control_teamviewer_remote_file_copy.toml:11]:
```suggestion
Identifies an executable or script file remotely downloaded via a TeamViewer transfer session.
```
Unless `verify the context of this activity` is expanded, I don't think it is needed.

**Review by bm11100 — APPROVED** [review]
one suggestion, otherwise LGTM

**Review by threat-punter — APPROVED** [review]

**Pull request merged**